ui/app/routes/vault/cluster/clients/index.js

```diff
@@ -1,85 +1,40 @@
 import Route from '@ember/routing/route';
-import ClusterRoute from 'vault/mixins/cluster-route';
```

Comment:
Okay when the heck do we need `import ClusterRoute from 'vault/mixins/cluster-route'` vs `import Route`

Reply:
When it's outside of `{ }` we're importing the default export, and so the name here is just whatever we want to call it

Reply:
But if it was breaking things then I have no idea 🤔